examples/research_projects/jax-projects/HOW_TO_PROPOSE_PROJECT.md

```diff
@@ -0,0 +1,97 @@
+If your project already exists - great! This means that you can comment and improve
+the existing idea and join the project to form a team! If your project idea already 
+exists for a different language, feel free to submit the same project idea, just in 
+a different language.
+Third, having ensured that your project doesn't exist, click on the *"New Topic"*
+button on the [Flax/JAX Projects Forum category](https://discuss.huggingface.co/c/flax-jax-projects/22) to create a new project proposal.
+Fourth, make sure that your project proposal includes the following information:
+
+- 1. *A clear description of the project*
+- 2. *In which language should the project be conducted?* English, German, Chinese, ...? It can also be a multi-lingual project
+- 3. *Which model should be used?* If you want to adapt an existing model, you can add the link to one of the 4000 available checkpoints in JAX [here](https://huggingface.co/models?filter=jax) If you want to train a model from scratch, you can simply state the model architecture to be used, *e.g.* BERT, CLIP, etc. You can also base your project on a model that is not part of transformers. For an overview of libraries based on JAX, you can take a look at [awesome-jax](https://github.com/n2cholas/awesome-jax#awesome-jax-). **Note** that for a project that is not based on Transformers it will be more difficult for the 🤗 team to help you. Also have a look at the section [Quickstart Flax & Jax in Transformers](https://github.com/huggingface/transformers/tree/master/examples/research_projects/jax-projects#quickstart-flax-and-jax-in-transformers) to see what model architecture are currently supported in 🤗 Transformers.
+- 4. *What data should be used?* It is important to state at least what kind of data you would like to use. Ideally you can already point to publicly available data or a dasaset that exists in the 🤗 Datasets library.
+- 5. *Are similar training scripts available in Flax/JAX?* It would be important to find similar training scripts that already exist in Flax/JAX. *E.g.* if you are working on a Seq-to-Seq task, you can make use of the [`run_summarization_flax.py`](https://github.com/huggingface/transformers/blob/master/examples/flax/summarization/run_summarization_flax.py) script which is very similar to any seq2seq training. Also have a look at the section [Quickstart Flax & Jax in Transformers](https://github.com/huggingface/transformers/tree/master/examples/research_projects/jax-projects#quickstart-flax-and-jax-in-transformers) to see what training scripts are currently supported in 🤗 Transformers.
+- 6. *(Optionally) What are possible challenges?*, List possible difficulties with your project. *E.g.* If you know that training convergence usually takes a lot of time, it is worth stating this here!
+- 7. *(Optionally) What is the desired project outcome?* - How would you like to submit the project? How would you like to demo your project? One could *e.g.* create a Streamlit application.
```

Comment:
This just "How would you like to demo your project" should be enough. Submission guidelines will be given by us, no ?

Reply:
yes!